blocks/library/gallery/block.js

```diff
@@ -113,6 +115,21 @@ class GalleryBlock extends Component {
 		} );
 	}
 
+	appendImageAttributes( currentImages, newImages ) {
```

Comment:
Minor: Personal opinion but is this function really useful as a separate method? I generally try to minimise the component methods if not necessary (repeated code, exposed functions, event handlers)...

Reply:
No, it is not useful as a separate method. Normally, I try to avoid implementing a function inside another function but in this case and given that it is a minor function I think it is ok. So I followed your advice 👍